packages/zmarkdown/__tests__/__snapshots__/latex.tests.js.snap

```diff
@@ -285,7 +290,7 @@ exports[`list 1`] = `
 `;
 
 exports[`math 1`] = `
-"A sentence ($S$) with \\\\textit{italic} and inline math ($C_L$) and $$b$$ another.
+"A sentence ($S$) with \\\\textit{italic} and inline math ($C_L$) and $b$ another.
```

Comment:
Not sure where this change comes from and if it's intentional.

Reply:
You're right, this is a bad behaviour, finally found it and solved it!